ruby-audit-logs-example/views/send_events.erb

```diff
@@ -1,102 +1,220 @@
-                <p>Start selling to enterprise customers with just a few lines of code.</p>
-                <p>Implement features like single sign-on in minutes instead of months.</p>
-            </div>
-            <div class="flex success-buttons">
-                <a href="https://workos.com/signup" target="_blank"><button class='button'>Get Started</button></a>
-                <a href="mailto:sales@workos.com?subject=WorkOS Sales Inquiry" target="_blank"><button
-                        class='button button-outline sales-button'>Contact
-                        Sales</button></a>
+
+    <div class="logged_in_div_right">
+        <div class="logged_in_nav">
+            <div class="flex">
+                <div>
+                    <a href="https://workos.com/docs" target="_blank"><button class='button nav-item'>Documentation</button></a>
+                </div>
+                <div>
+                    <a href="https://workos.com/docs/reference" target="_blank"><button class='button nav-item'>API
+                        Reference</button></a>
+                </div>
+                <div>
+                    <a href="https://workos.com/blog" target="_blank"><button
+                        class='button nav-item blog-nav-button'>Blog</button></a>
+                </div>
+                <div>
+                    <a href="https://workos.com/" target="_blank">
+                    <img src="<%= url("images/workos_favicon.png")%>" class="nav-image"  alt="link to workos.com logo">
+                    </a> 
+                </div>
             </div>
         </div>
-        <div class="logged_in_div_right">
-            <div class="flex flex-start width-40vw">
-                <div><h1><%=@org_name%></h1></div>
+        <div class="flex_column width-65vw logged_in_right_content">
+            <div class="flex flex-start width-65vw page-title">
+                <div><h2>Audit Logs</h2></div>
             </div>
-            <div class="flex_column width-40vw ">
-                <div class="flex card width-40vw space-between ">
-                    <code class="org-id"><%=@organization_id%></code>
+            <div class="flex card width-65vw space-between ">
+                <code class="org-id"><%= @organization.id %></code>
+                <div>                    
                     <a href="/logout"><button class="button">Change Org</button></a>
-                </div>
-                <div class="card width-40vw">
-                    <div class="flex space-evenly width-40vw margin-b-25">
-                        <div>
-                            <div class="button button-outline width-15vw">
-                                <a href="https://workos.com/docs/audit-logs" target="_blank"><p>Audit Logs Docs</p></a>                            
-                            </div>                    
+                </div>                
+            </div>
+            <div class="card events-card width-65vw height-45vh">
```

Comment:
Can you remove the height-45vh class here? It's causing the card to chop off the content. 
![CleanShot 2023-03-27 at 14 14 17](https://user-images.githubusercontent.com/47123183/228056059-4d9253b2-3358-4e02-b34c-bb92b602fbd3.gif)